core/server/api/index.js

```diff
@@ -209,6 +209,10 @@ http = function http(apiMethod) {
             // Add X-Cache-Invalidate, Location, and Content-Disposition headers
             return addHeaders(apiMethod, req, res, (response || {}));
         }).then(function then(response) {
+            if (req.method === 'DELETE') {
+                return res.status(204).end();
```

Comment:
I'm at a computer now so looking at this a little more closely. If you use `sendStatus`, the body does get set to `No Content`, but later on `res.send` checks for 204s and strips the body.

So I'm torn between leaving it as and explicitly leaving the response body empty, or using `sendStatus`, which sets and then strips the body later (which seems a little silly).

If you have a strong preference to `sendStatus` I'm cool with changing it, otherwise maybe I'll just leave it as is?